packages/fiori/src/UploadCollection.ts

```diff
@@ -12,7 +12,7 @@ import List from "@ui5/webcomponents/dist/List.js";
 import ListMode from "@ui5/webcomponents/dist/types/ListMode.js";
 import Title from "@ui5/webcomponents/dist/Title.js";
 import IllustratedMessage from "./IllustratedMessage.js";
-import "./illustrations/sapIllus-Scene-Tent.js";
```

Comment:
Unfortunately, the way the illustrations registry is implemented now, only the top level import "Tent.js" registers an illustration by its name. The specific module sapIllus-Scene-Tent only exports the SVG content without registering an illustration. Later when the framework tries to fetch an illustration to display it, looks into that registry. 
This is something that could be improved.
For example, the icons are handled smarter. There, we have top level import to an icon that registers both v4 or v5(Horizon) icon, but you can also import directly the "v4"(Quartz) or "v5"(Horizon) icon and display it. Same can happen here - more fine granular modules for the specific illustration sizes could be provided. For now, we should import "Tent.js" and hardcode "Scene" in the template, if this was the idea.